src/code/views/app-view.coffee

```diff
@@ -44,26 +44,28 @@ module.exports = React.createClass
   render: ->
     (div {className: 'app'},
       (div {className: if @state.iframed then 'iframed-workspace' else 'workspace'},
-        if not @state.iframed
+        if not @state.iframed && AppSettingsStore.store.settings.uiElements.globalNav != false
           (GlobalNav
             filename: @state.filename
             username: @state.username
             graphStore: @props.graphStore
             GraphStore: @GraphStore
+            display: AppSettingsStore.store.settings.uiElements.globalNav
           )
-        (div {className: 'action-bar'},
+        (div {className: if AppSettingsStore.store.settings.uiElements.actionBar == false then 'action-bar hidden' else if AppSettingsStore.store.settings.uiElements.globalNav is false then 'action-bar small' else 'action-bar'},
```

Comment:
&^&^$# CoffeeScript! 

I find this line hard to read.  Could you extract a variable here `actionBarStyle` on line 45 or so, and use `if` `else` indenting?   

I have seen people use this style